release_notes/ocp-4-6-release-notes.adoc

```diff
@@ -2541,3 +2514,14 @@ Issued: 2020-11-09
 Space precluded documenting all of the container images for this release in the advisory. See the following article for notes on the container images in this release:
 
 link:https://access.redhat.com/solutions/5551151[{product-title} 4.6.3 container image list]
+
+[id="ocp-4-6-3-bug-fixes"]
+==== Bug fixes
+
+* Due to a known issue, the GPU Operator and Node Feature Discovery (NFD) Operator were not available on a fresh installation of {product-title} 4.6.1. You were required to install {product-title} 4.5 and upgrade the cluster to version 4.6.1 to use the GPU and NFD Operators. This issue has been fixed, and the GPU and NFD Operators are now available on a fresh installation of {product-title} 4.6.3 and later. link:https://bugzilla.redhat.com/show_bug.cgi?id=1890673[*BZ#1890673*].
```

Comment:
I don't think there's normally a period after the BZ link

Reply:
Ah, thank you :sweat_smile: